ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge2.cfg

```diff
@@ -0,0 +1,368 @@
+         template interval 5000
+   no shutdown
+!
+transceiver qsfp default-mode 4x10G
+!
+service routing protocols model multi-agent
+!
+hostname cv-pathfinder-edge2
+!
+router adaptive-virtual-topology
+   topology role edge
+   region AVD_Land_West id 42
+   zone DEFAULT-ZONE id 1
+   site Site423 id 423
+   !
+   policy DEFAULT-AVT-POLICY
+      !
+      match application-profile VIDEO
+         avt profile DEFAULT-AVT-POLICY-VIDEO
+      !
+      match application-profile default
+         avt profile DEFAULT-AVT-POLICY-DEFAULT
+   !
+   policy DEFAULT-AVT-POLICY-WITH-CP
+      !
+      match application-profile CONTROL-PLANE-APPLICATION-PROFILE
+         avt profile CONTROL-PLANE-PROFILE
+      !
+      match application-profile VIDEO
+         avt profile DEFAULT-AVT-POLICY-VIDEO
+      !
+      match application-profile default
+         avt profile DEFAULT-AVT-POLICY-DEFAULT
+   !
+   policy PROD-AVT-POLICY
+      !
+      match application-profile VOICE
+         avt profile PROD-AVT-POLICY-VOICE
+      !
+      match application-profile VIDEO
+         avt profile PROD-AVT-POLICY-VIDEO
+      !
+      match application-profile default
+         avt profile PROD-AVT-POLICY-DEFAULT
+   !
+   profile CONTROL-PLANE-PROFILE
+      path-selection load-balance LB-CONTROL-PLANE-PROFILE
+   !
+   profile DEFAULT-AVT-POLICY-DEFAULT
+      path-selection load-balance LB-DEFAULT-AVT-POLICY-DEFAULT
+   !
+   profile DEFAULT-AVT-POLICY-VIDEO
+      path-selection load-balance LB-DEFAULT-AVT-POLICY-VIDEO
+   !
+   profile PROD-AVT-POLICY-DEFAULT
+      path-selection load-balance LB-PROD-AVT-POLICY-DEFAULT
+   !
+   profile PROD-AVT-POLICY-VIDEO
+      path-selection load-balance LB-PROD-AVT-POLICY-VIDEO
+   !
+   profile PROD-AVT-POLICY-VOICE
+      path-selection load-balance LB-PROD-AVT-POLICY-VOICE
+   !
+   vrf default
+      avt policy DEFAULT-AVT-POLICY-WITH-CP
+      avt profile DEFAULT-AVT-POLICY-DEFAULT id 1
+      avt profile DEFAULT-AVT-POLICY-VIDEO id 3
+      avt profile CONTROL-PLANE-PROFILE id 254
+   !
+   vrf IT
+      avt policy DEFAULT-AVT-POLICY
+      avt profile DEFAULT-AVT-POLICY-DEFAULT id 1
+      avt profile DEFAULT-AVT-POLICY-VIDEO id 3
+   !
+   vrf PROD
+      avt policy PROD-AVT-POLICY
+      avt profile PROD-AVT-POLICY-DEFAULT id 1
+      avt profile PROD-AVT-POLICY-VOICE id 2
+      avt profile PROD-AVT-POLICY-VIDEO id 4
+!
+router path-selection
+   tcp mss ceiling ipv4 ingress
+   !
+   path-group INET id 101
+      ipsec profile CP-PROFILE
+      !
+      local interface Ethernet1
+         stun server-profile INET-cv-pathfinder-pathfinder-Ethernet1 INET-cv-pathfinder-pathfinder-Ethernet3
+      !
+      peer dynamic
+      !
+      peer static router-ip 192.168.144.1
+         name cv-pathfinder-pathfinder
+         ipv4 address 10.7.7.7
+         ipv4 address 10.9.9.9
+   !
+   path-group LAN_HA id 65535
+      ipsec profile CP-PROFILE
+      flow assignment lan
+      !
+      local interface Ethernet52
```

Comment:
I think it will be beneficial to have multiple uplink interfaces to see how it maps together.

Reply:
done!